server/src/main/java/org/apache/druid/segment/loading/SegmentLoaderConfig.java

```diff
@@ -56,6 +56,9 @@
   @JsonProperty("numBootstrapThreads")
   private Integer numBootstrapThreads = null;
 
+  @JsonProperty("numCacheLoadThreads")
+  private int numCacheLoadThreads = 5;
```

Comment:
What is the purpose to make it a configuration item ? I think if we can make it default to the number of CPU cores? Or can we re-use `numBootstrapThreads` because it is defined as:
> How many segments to load concurrently during historical startup.